Protobuild.toml

```diff
@@ -0,0 +1,33 @@
+version = "unstable"
+generator = "gogoswarm"
+plugins = ["grpc", "deepcopy", "storeobject", "raftproxy", "authenticatedwrapper"]
+
+# Control protoc include paths. Below are usually some good defaults, but feel
+# free to try it without them if it works for your project.
+[includes]
+  # Include paths that will be added before all others. Typically, you want to
+  # treat the root of the project as an include, but this may not be necessary.
+  before = [".", "./protobuf"]
```

Comment:
Let's go ahead and use the full include path so we don't need these extra paths. `plugin/plugin.proto` doesn't provide as much information as `github.com/docker/swarmkit/protobufplugin/plugin.proto`.